samples/addons/prometheus.yaml

```diff
@@ -66,8 +66,6 @@ data:
       kubernetes_sd_configs:
       - role: node
       relabel_configs:
-      - action: labelmap
```

Comment:
this is generated from helm input values in `manifests/addons/values-prometheus.yaml`.

I am not against changing these, but the (original) intent was to have a minimally modified prometheus deployment so that (1) we work out of the box (2) if users want tweaks for improvements they can follow our values.yaml we used to their own install and do so easily. So if we want to make this change it would be nice if its possible to express it in terms of values.yaml inputs to the upstream chart and note this is an optimization only, not required (right?)

Reply:
100% fine with the operator one, by the way

Reply:
not required. forgot this was generated (was goofing around during the TOC meeting). I'll take a look at the helm now.